$NetBSD: patch-ad,v 1.7 2010/06/14 10:35:13 gregoire Exp $

Description: forces installation of all configuration scripts.  These are
  installed in share/examples/grub2.  The appropriates ones (as in the
  original behavior below) are copied into etc/grub2 with the pkgsrc
  directive CONF_FILES.  See the main makefile of this pkgsrc package for
  further details.

--- conf/common.rmk.orig	2010-01-11 02:40:56.000000000 +0100
+++ conf/common.rmk
@@ -202,9 +202,9 @@ CLEANFILES += grub-gettext_lib
 	./config.status --file=$@:$<
 	chmod +x $@
 grub-mkconfig_SCRIPTS = 00_header 30_os-prober 40_custom 41_custom
-ifneq (, $(host_kernel))
-grub-mkconfig_SCRIPTS += 10_$(host_kernel)
-endif
+#ifneq (, $(host_kernel))
+grub-mkconfig_SCRIPTS += 10_hurd 10_kfreebsd 10_linux 10_netbsd 10_windows
+#endif
 
 CLEANFILES += $(grub-mkconfig_SCRIPTS)
 
